Context/ConfiguredContext.php

```diff
@@ -60,6 +60,9 @@ class ConfiguredContext implements SearchContextInterface
      */
     private $interceptorProvider;
 
+    /** @var ContextSortAttribute */
```

Comment:
That or use a local static variable within the private method if you're worried about too many object instantiations